src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -0,0 +1,479 @@
+   *
+   */
+  values: string[];
+
+  /**
+   * Specifies the index field whose values the `SimpleFilter` should use.
+   *
+   * Specifying a value for this option is required for the `SimpleFilter` component to work.
+   */
+  field: IFieldOption;
+
+  /**
+   * Specifies a JSON object describing a mapping of SimpleFilter values to their desired captions.
+   * **Examples:**
+   *
+   * You can set the option in the ['init']{@link init} call:
+   * ```javascript
+   * var myValueCaptions = {
+   *   "txt" : "Text files",
+   *   "html" : "Web page",
+   *   [ ... ]
+   * };
+   *
+   * Coveo.init(document.querySelector("#search"), {
+   *   SimpleFilter : {
+   *     valueCaption : myValueCaptions
+   *   }
+   * });
+   * ```
+   *
+   * Or before the `init` call, using the ['options']{@link options} top-level function:
+   * ```javascript
+   * Coveo.options(document.querySelector("#search"), {
+   *   SimpleFilter : {
+   *     valueCaption : myValueCaptions
+   *   }
+   * });
+   * ```
+   *
+   * Or directly in the markup:
+   * ```html
+   * <!-- Ensure that the double quotes are properly handled in data-value-caption. -->
+   * <div class='CoveoSimpleFilter' data-field='@myotherfield' data-value-caption='{"txt":"Text files","html":"Web page"}></div>
+   * ```
+   */
+  valueCaption: any;
+
+  /**
+   * Specifies the macimum number of field values to display by default in the SimpleFilter. Default value is 5.
+   */
+  maximumNumberOfValues: number;
+
+}
+/**
+ * The `ILabeledCheckbox` interface describes a custom object type that is used to build checkboxes and their label, using the standard `string`
+ * type and the [ `Checkbox` ]{@link Checkbox} class. This is used to build the checkboxes used by the [ `SimpleFilter` ]{@link SimpleFilter} component.
+ */
+interface ILabeledCheckbox {
+  checkbox: Checkbox;
+  label: string;
+}
+
+/**
+ * The `SimpleFilter` component displays a dropdown menu containing values used to filter the results returned by queries.
+ *
+ * The list of values is either static (defined by the user through the [ `values` ]{@link ISimpleFilterOptions.values} option
+ * or dynamic (obtained automatically using a [ `GroupByRequest` ]{@link IGroupByRequest} operation performed at the same time as the main query).
+ */
+export class SimpleFilter extends Component {
+  static ID = 'SimpleFilter';
+  static doExport = () => {
+    exportGlobally({
+      'SimpleFilter': SimpleFilter
+    });
+  }
+
+  static options: ISimpleFilterOptions = {
+    maximumNumberOfValues: ComponentOptions.buildNumberOption({ defaultValue: 5 }),
+    values: ComponentOptions.buildListOption<string>(),
+    field: ComponentOptions.buildFieldOption({ required: true }),
+    title: ComponentOptions.buildStringOption({ defaultValue: l('NoTitle') }),
+    valueCaption: ComponentOptions.buildJsonOption()
+  };
+
+  /**
+   * The element containing all of the checkboxes in the component.
```

Comment:
It used to be a public property, I noticed I left the doc there after pushing. My bad!